droidlet/perception/robot/active_vision/candidate_selection.py

```diff
@@ -81,15 +84,24 @@ def is_good_candidate(self, x):
             return False
         
         return True
-        
-    def get_n_candidates(self, n, good=True):
+    
+    def get_n_candidates(self, n, good, evenly_spaced=False):
         # go through the images and filter candidates
         # mark all the good candidates and then uniformly sample from them 
         # Pick n things uniformly from all the good candidates
+        def sample_even_or_random(arr, n, evenly_spaced=False):
```

Comment:
out of curiosity, why the internal function?  why not do directly in the switch in 96-98?  not asking for a change just want to understand the style